src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/MinimumOSAttribute.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.InteropServices
+{
+
+    /// <summary>
+    /// Records the minimum platform that is required in order to the marked thing.
```

Comment:
```suggestion
    /// Records the minimum platform that is required in order to [something goes here] the marked thing.
```

:smile:

Reply:
And we should use something other than "the marked thing", e.g. "the attributed member".